File: algorithm-programmers-sort-h-index/src/test/java/com/currenjin/SolutionTest.java
package com.currenjin;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class SolutionTest {
    @Test
    void sample1() {
        assertEquals(3, new Solution().solution(new int[]{3, 0, 6, 1, 5}));
    }

    @Test
    void edgeCase() {
        assertEquals(4, new Solution().solution(new int[]{10, 8, 5, 4, 3}));
    }
}

File: algorithm-programmers-sort-h-index/src/test/kotlin/com/currenjin/SolutionTest.kt
package com.currenjin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class SolutionTest {
    @Test
    fun sample1() {
        assertEquals(3, Solution().solution(intArrayOf(3, 0, 6, 1, 5)))
    }

    @Test
    fun edgeCase() {
        assertEquals(4, Solution().solution(intArrayOf(10, 8, 5, 4, 3)))
    }
}
